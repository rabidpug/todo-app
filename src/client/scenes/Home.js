import React from 'react';

const Home = () => <div>Welcome to React BoilerPlate!</div>;

export default Home;
