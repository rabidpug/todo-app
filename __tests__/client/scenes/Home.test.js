import Home from 'Scenes/Home';
import React from 'react';
import { shallow, } from 'enzyme';
import { shallowToJson, } from 'enzyme-to-json';

describe(
  'TopBar', () => {
    it(
      'Should render the TopBar', () => {
        const wrapper = shallow( <Home /> );

        expect( shallowToJson( wrapper ) ).toMatchSnapshot();
      }
    );
  }
);
