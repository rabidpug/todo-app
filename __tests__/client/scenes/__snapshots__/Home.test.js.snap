// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Should render the TopBar 1`] = `
<div>
  Welcome to React BoilerPlate!
</div>
`;
